packages/web-react/src/components/UNSTABLE_Truncate/stories/UNSTABLE_Truncate.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { Markdown } from '@storybook/blocks';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import ReadMe from '../README.md';
+import { UNSTABLE_Truncate } from '..';
+
+const meta: Meta<typeof UNSTABLE_Truncate> = {
+  title: 'Experimental/UNSTABLE_Truncate',
+  component: UNSTABLE_Truncate,
+  parameters: {
+    docs: {
+      page: () => <Markdown>{ReadMe}</Markdown>,
+    },
+  },
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    lines: {
+      control: 'number',
```

Comment:
please add a line with the default value. See other stories on how to do it. (it is something like table default value or sth)

Reply:
nope, the default value, as in CSS is 3